File: C++/ejercicio2.cpp
#include <iostream>

using namespace std;

int main () {
	
	cout << "Hola!, ¿Como te llamas? ";
	
	char nombre[20];
	cin >> nombre;
	cout << "Estoy encantado de conocerte " << nombre;

	cout << "Cuantos años tienes?";
	float edad;
	cin >> edad;
	cout << "Dentro de 10 años tendras " << edad + 10;
	
	return 0;
	
}

File: Java/ejercicio2.java
import java.util.Scanner;

public class ejercicio2 {

	public static void main(String[] args) {
		
		System.out.println("Hola!, ¿Como te llamas? ");
		Scanner n = new Scanner(System.in);
		String nombre = n.nextLine();
		System.out.println("Estoy encantado de conocerte " + nombre);
		
		System.out.println("¿Cuantos años tienes? ");
		Scanner e = new Scanner(System.in);
		int edad = e.nextInt();
		System.out.println("Dentro de 10 años tendras " + (edad + 10));
		
		
	}
}